vmdb/app/models/service_template.rb

```diff
@@ -19,7 +19,10 @@ class ServiceTemplate < ActiveRecord::Base
   belongs_to :service_template_catalog
 
   virtual_has_many :custom_buttons
-  virtual_column   :type_display, :type => :string
+  virtual_column   :type_display,                 :type => :string
+  virtual_column   :template_valid,               :type => :boolean
+  virtual_column   :template_valid_error_message, :type => :string
+  alias template_valid template_valid?
```

Comment:
This line belongs after the `template_valid?` method.  See the archived? method as an example [here](https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/models/vm_or_template.rb#L1294).